packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -80,17 +85,27 @@ class EnumerationWidget extends React.Component {
 			onClick: this.onDeleteItem.bind(this),
 		}];
 		this.defaultHeaderActions = [{
-			label: 'Add item',
-			icon: 'talend-plus',
-			id: 'add',
-			onClick: this.changeDisplayToAddMode.bind(this),
-		}, {
 			disabled: false,
 			label: 'Search for specific values',
 			icon: 'talend-search',
 			id: 'search',
 			onClick: this.changeDisplayToSearchMode.bind(this),
+		}, {
+			label: 'Add item',
+			icon: 'talend-plus',
+			id: 'add',
+			onClick: this.changeDisplayToAddMode.bind(this),
 		}];
+
+		if (this.allowImport) {
+			this.defaultHeaderActions.splice(1, 0, {
+				label: 'Import values from a file',
+				icon: 'talend-download',
+				id: 'upload',
+				onClick: this.simulateClickInputFile.bind(this),
```

Comment:
Why do you use an intermediate method instead of using `this.inputImport.click.bind(this)` directly ?